yearn/contracts/CurveVoterProxyStrategy.sol

```diff
@@ -328,6 +328,26 @@ contract CurveVoterProxyStrategy is BaseStrategy {
             tbtcCurvePoolGauge,
             address(want)
         );
+        // Harvest the CRV rewards from the Curve pool's gauge and transfer
```

Comment:
Oh, that's a good one. We do transfer `crv` & `cvx` + extra rewards in convexStrategy. Perhaps we missed it here? I think Łukasz will shed more light on it.

Reply:
Yep indeed. Good catch Tomasz!